src/app/reducers/conceptMapper.ts

```diff
@@ -133,9 +133,7 @@ function mapCsvTextToConcept(
       harTelefon: mapToSingleValue(csvMap, 'kontaktpunkt_telefon')
     },
     status: mapToSingleValue(csvMap, 'status') ?? ConceptStatus.UTKAST,
-    tildeltBruker: {
-      id: mapToSingleValue(csvMap, 'tildeltBruker_id') ?? ''
-    }
+    assignedUser: mapToSingleValue(csvMap, 'assignedUser') ?? ''
```

Comment:
Tror vi bare må fjerne støtte for feltet ved import fra csv, regner med vi vil få en del diverse der når vi egentlig kun støtter id fra den interne users-lista

Reply:
Fjernet